CHANGELOG.txt

```diff
@@ -2,6 +2,17 @@
 CHANGELOG
 =========
 
+-------------------------------------------------------------------------------
+Nov, 16, 2021 1.2.1
+-------------------------------------------------------------------------------
+
+Minor:
+- Raise the value error when min_frequency is a percentage, while it is set to
+  be less than 1/(Number of sequences).
```

Comment:
> you mean less than or equal?

Equal could be ok since it will allow the minimum threshold of row count to be 1. So when it is an equal, it is not throwing an error.